lessons/intro/distribution/index.md

```diff
@@ -1,9 +1,10 @@
-moduly
+Moduly
 ======
 
 Zatím jsme tvořili programy v Pythonu tak nějak na divoko, tedy v jednom nebo
-více souborech bez nějakého zvláštního řádu. Na dnešním cvičení se podíváme na
-to, jak tvořit redistribuovatelné moduly, které jdou instalovat pomocí pipu.
+více souborech bez nějakého zvláštního řádu. Dnes se podíváme na
```

Comment:
Pokud má text fungovat i pro čtenáře zvenčí, dal bych místo "dnes" "v této lekci".